requirements.txt

```diff
@@ -1,17 +1,18 @@
-wxPython==4.0.1
+wheel==0.30.0
+wxPython==4.1.1
 docker==2.6.1
 construct==2.8.14
 mne==0.17.0
 opencv_python==4.1.0.25
-PsychoPy==2020.2.5
-pyglet==1.5.8
+PsychoPy==2020.2.10
+pyglet==1.4.10
 numpy==1.19.2
 sounddevice==0.4.1
 SoundFile==0.10.3.post1
 scipy==1.5.2
 scikit-learn==0.23.2
 seaborn==0.9.0
-matplotlib==2.1.1
+matplotlib==3.1.1
```

Comment:
Did you test to see if the signal viewer still works with this version?

Reply:
Good call. This upgrade didn't impact the viewer, but the WxPython one did. I'll look into it more tomorrow.